scvi/model/base/_base_model.py

```diff
@@ -346,8 +346,8 @@ def load(
 
         Examples
         --------
-        >>> vae = SCVI.load(save_path, adata)
-        >>> vae.get_latent_representation()
+        >>> model = ModelClass.load(save_path, adata)
+        >>> model.get_....
```

Comment:
Personally I would either remove the Examples section altogether or call out this example is specifically for the SCVI class.
This is also fine, it just does not add too much value than if we had no Examples IMO.

Reply:
I added a comment